src/Ombi/ClientApp/app/requests/movierequests.component.ts

```diff
@@ -175,11 +175,18 @@ export class MovieRequestsComponent implements OnInit {
         });
     }
 
-    public setOrder(value: string) {
+    public setOrder(value: string, el: any) {
+        el = el.toElement || el.relatedTarget || el.target || el.srcElement;
+
+        const parent = el.parentElement;
+        const previousFilter = parent.querySelector(".active");
+
         if (this.order === value) {
-          this.reverse = !this.reverse;
+            this.reverse = !this.reverse;
+        } else {
+            previousFilter.className = "";
+            el.className = "active";
```

Comment:
This function receives the event as a parameter gets the element that the click occurred from and the parent element. We query for whether an element is active and if its not the element where the click occurred we remove the active class from the default or previous element and enable the active class on the element where click occurred.  This is for styling currently active filter in the movie requests filter option.